src/app/[locale]/_utils/__tests__/storage-utils.test.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+    storageDelete,
+    storageRead,
+    storageWrite,
+    storageOpsBuilder,
+    StorageOps,
+  } from './../storage-utils';
+  
+  describe('Storage Functions', () => {
+    const key = 'testKey';
+    const value = 'testValue';
+  
+    beforeEach(() => {
+      localStorage.clear();
+      sessionStorage.clear();
+    });
+  
+    test('storageDelete deletes the correct key', () => {

```

Comment:
As above.  For example: "storageDelete should delete the item corresponding to the passed key". Just to be consistent with jest wording. Even without "should" (if we want to follow the [spotify philosophy](https://github.com/spotify/should-up), but then we must be consistent in all our description. To me, I like "should". BTW, I think that having more verbose description helps. What do you think about that?

Reply:
You're right, let's create a more verbose description then. I also prefer using 'should' :)